mentii/mentii
# UX changes

Changed language in text and problem solving for UX improvement.

**Significant Changes**
1. Added mentii avatar and changed problem statement to be like a student asking for help
2. Buttons are less assertive and have icons and language that focus on helping Mentii
3. When a button reaches a solution (even if it's a bad step solution) the button changes to **Confirm solution**
4. The non-current correction steps have a lighter red color
5. Changed toast messages to be directed to helping Mentii

**How to Test**
1. Go to do a problem and step through things
2. Go through bad step paths (see note below)

**Notes**
Check that the UX looks the same for when going down a bad step path. This will required disabling the dropdown and choosing a step limit 3 or &infin;
